pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -124,14 +125,15 @@ def setUpClass(cls):
         cls.tasks_api = TasksApi(core_client)
         cls.migration_plans_api = MigrationPlansApi(migration_client)
 
-        set_pulp2_snapshot(name='20191031')
+        set_pulp2_snapshot(name='file_base_4repos')
 
     def tearDown(self):
         """
         Clean up the database after each test
         """
         cmd = get_psql_smash_cmd(TRUNCATE_TABLES_QUERY_BASH)
         self.smash_cli_client.run(cmd, sudo=True)
+        time.sleep(0.5)
```

Comment:
Any better ideas how to wait for the truncate operation to finish?
Without this sleep, I was getting `deadlock detected` in postgres every now and then.

Reply:
Nope : /